unit-tests/heitech.zer0mqXt.core.tests/ConfigurationTestData.cs

```diff
@@ -27,5 +28,11 @@ public IEnumerator<object[]> GetEnumerator()
         }
 
         IEnumerator IEnumerable.GetEnumerator() => GetEnumerator();
+
+
+        public static ISocket BuildInprocSocketInstanceForTest(string pipeName)
+        {
+            return Zer0Mq.Go().SilenceLogger().BuildWithInProc(pipeName);
+        }
```

Comment:
capitalization mismatch:
- the method's name has `...Inproc...`
- but the `IZer0MqBuilder` has `...InProc`

which is the "right" capitalization?

Reply:
InProc is the correct spelling as it is used widely this way.